.github/workflows/gh-action-weblate-upload.yml

```diff
@@ -0,0 +1,28 @@
+# This GitHub action updates the translation (msgmrge) based on the changed source language file
+
+name: Update Weblate translations
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "addons/resource.language.en_gb/resources/strings.po"
+      - ".github/workflows/gh-action-weblate-upload.yml"
+jobs:
+  weblate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 0
+      - name: Update Weblate
+        env:
+          TOKEN: "${{ secrets.WEBLATE_TOKEN }}"
+        run: | 
+          curl -v \
```

Comment:
please remove the -v as it will show the auth token in logs

Reply:
Thanks, will do.